index.html

```diff
@@ -0,0 +1,215 @@
+                                </p>
+                            </li>
+
+                        </li>
+
+                    </ul>
+
+                </div>
+
+                <div class="bg">
+                    <ul class="values-reasons">
+                        <li>
+                            <img
+                                src="/first-capstone-project/assets/images/enriching.png" />
+                            <li><p>Enriching</p></li>
+                            <li>
+                                <p>
+                                    bettering lives through education and
+                                    other avenues
+                                </p>
+                            </li>
+
+                        </li>
+
+                    </ul>
+
+                </div>
+
+            </div>
+            <button type="button" class="donate-btn">Donate</button>
+
+            <div class="donate">
+                <img src="/first-capstone-project/assets/images/donate.png" />
+            </div>
+
+        </section>
+        <section class="projects">
+            <h3>Upcoming Events</h3>
+            <hr class="title-hr" />
+            <div class="events-list"></div>
+            <div class="more-btn" id="more-btn">
+                <p class="more">MORE</p>
+                <i class="arrow down"></i>
+
+            </div>
+            <div class="more-btn less-btn" id="less-btn">
+                <p class="more">LESS</p>
+                <i class="arrow up"></i>
+
+            </div>
+        </section>
+        <section class="partners">
+            <h3>Partners</h3>
+            <hr>
+            <div class="partners-desktop">
+                <div class="partners-logo">
+                    <img src="/first-capstone-project/assets/images/google.svg"
+                        alt="Google" />
+                    <img src="/first-capstone-project/assets/images/mozilla.svg"
+                        alt="Mozilla" class="mozilla" />
+                    <img src="/first-capstone-project/assets/images/airbnb.png"
+                        alt="Google" />
+
+                </div>
+                <div class="partners-logo">
+                    <div class="naver">
+                        <img
+                            src="/first-capstone-project/assets/images/naver.png"
+                            alt="Naver" />
+                    </div>
+
+                    <img src="/first-capstone-project/assets/images/unhcr.svg"
+                        alt="Mozilla" class="mozilla" />
+
+                </div>
+            </div>
+
+        </section>
+        <footer>
+            <div class="copyright">
+                <h3>
+                    @copyright
+                </h3>
+                <p>Linc Technologies</p>
+
+            </div>
+            <div class="copyright-text">
+                <p>This applications is property of...</p>
+
+            </div>
+        </footer>
```

Comment:
Good job by using semantic html tags to structure the page 👍🏼  👍🏼  👍🏼 , one smathing to mention is the background color it is white in the home page refere to the images 

![Capture d’écran du 2023-08-01 23-59-39](https://github.com/NoelLincoln/first-capstone-project/assets/59707859/8ea488b6-60e3-4f4e-8b19-9bf150353902)
![Capture d’écran du 2023-08-01 23-58-35](https://github.com/NoelLincoln/first-capstone-project/assets/59707859/1df98dc3-81c4-4591-bf52-ca7161b3a3a0)